### PR TITLE
Remove reference to first-network

### PR DESCRIPTION
**Pull request by nikhil550**
Signed-off-by: Nikhil Gupta <ngupta@symbridge.com>

#### Type of change

- Documentation update

#### Description

With the BYFN long gone, and past the LTS release, we can probably remove this reference to the first-network tutorial

### CODE REVIEW

File: docs/source/test_network.md [hunk 1]
```diff
@@ -6,8 +6,7 @@ can deploy a test network by using scripts that are provided in the
 by running nodes on your local machine. More experienced developers can use the
 network to test their smart contracts and applications. The network is meant to
 be used only as a tool for education and testing. It should not be used as a
-template for deploying a production network. The test network is being introduced
-in Fabric v2.0 as the long term replacement for the `first-network` sample.
+template for deploying a production network.
```
Comment (joealewine) [inline docs/source/test_network.md]: Might be relevant to add a link to the deployment guide here instead?

Comment (nikhil550) [reply docs/source/test_network.md]: Good idea. Added

**Review by joealewine — APPROVED** [review]

**Comment by nikhil550** [comment]
@mergifyio backport release-2.3

**Comment by nikhil550** [comment]
@Mergifyio backport release-2.2

**Pull request merged**